src/homepage/ForDataUsers/ResearchAndReports.jsx

```diff
@@ -4,13 +4,14 @@ export const ResearchAndReports = ({ isProdBeta }) => {
   if (isProdBeta) return null
 
   return (
-    <header>
-      <h3>
-        <ExternalLink url='https://www.consumerfinance.gov/data-research/hmda/'>
-          Research and Reports
-        </ExternalLink>
-      </h3>
+    <article>
+      <h3>Research and Reports</h3>
       <p>Research and reports on mortgage market activity.</p>
-    </header>
+      <ul>
+        <li>
+          <a href="https://www.consumerfinance.gov/data-research/hmda/" target="_blank">Research and Reports</a>
```

Comment:
Before there was an external icon to let users know they were going to leave the site, can we re-add that?

Replace the `a` tag with...

```js
<ExternalLink url='https://www.consumerfinance.gov/data-research/hmda/'>
  Research and Reports
</ExternalLink>
```

Reply:
Fixed.